operations/iuf/stages/management_nodes_rollout.md

```diff
@@ -26,22 +28,89 @@ The following arguments are most often used with the `management-nodes-rollout`
-| Limit management rollout list            | `--limit-management-rollout LIMIT_MANAGEMENT_ROLLOUT` | List of NCN management nodes to be rolled out, specified by HSM role and subrole       |
+| Concurrent management rollout percentage | `-cmrp CONCURRENT_MANAGEMENT_ROLLOUT_PERCENTAGE`      | Percentage value that limits the number of NCN worker nodes rolled out in parallel |
+| Limit management rollout list            | `--limit-management-rollout LIMIT_MANAGEMENT_ROLLOUT` | List of NCN management nodes to be rolled out, specified by HSM role and subrole (Management_Master, Management_Worker)       |
 
 ## Execution details
 
 The code executed by this stage exists within IUF. See the `management-nodes-rollout` entry in `/usr/share/doc/csm/workflows/iuf/stages.yaml` and the corresponding files in `/usr/share/doc/csm/workflows/iuf/operations/`
 for details on the commands executed.
 
-There are two methods administrators can use to avoid rollouts on specific NCN management nodes: by specifying the `--limit-management-rollout` argument or by using `kubectl` to label nodes with `iuf-prevent-rollout=true`.
+There are two methods administrators can use to avoid rollouts on specific NCN management nodes: by specifying the `--limit-management-rollout` argument for specifying which group(s) of nodes should be rebuilt or by using `kubectl` to label nodes with `iuf-prevent-rollout=true`.
 
-When NCN management nodes are being rebuilt, an additional Argo workflow will execute in addition to the standard IUF Argo workflows. It will include the string `ncn-lifecycle-rebuild` in its name.
+When NCN worker nodes are being rebuilt, an additional Argo workflow will execute in addition to the standard IUF Argo workflows. It will include the string `ncn-lifecycle-rebuild` in its name.
 
-## Example
+The `cmrp` parameter limits the percentage of worker nodes rolled out in parallel. The worker node rebuild can coordinate rebuilding multiple worker nodes at once. It starts by rebuilding one worker node. Once the node has been removed from the cluster, the workflow checks if it safe to rebuild the next worker node based on what services are running in the cluster. If it is safe, it will proceed in rebuilding the next node (parts of this will be in parallel with the first worker node rebuild). If it is unsafe to rebuild becuase the cluster could get into a bad state, then it waits to rebuild the second node. The `cmrp` parameter selects the percentage of worker nodes that the worker node rebuild should coordinate rebuilding at one time. For example, if there are 15 worker nodes and `cmrp=33`, then 5 worker nodes will be rebuilt in one worker node rebuild workflow. The worker node rebuild workflow will rebuild each of the 5 worker nodes and it may rebuild some in parallel depending on if it is safe to do so.
```

Comment:
```suggestion
The `-cmrp` argument limits the percentage of worker nodes rolled out in parallel. The worker node rebuild can coordinate rebuilding multiple worker nodes at once. It starts by rebuilding one worker node. Once that node has been removed from the system, the workflow checks if it is safe to rebuild the next worker node based on what services are running in the system. If it is safe, it will proceed to rebuild the next node, partially in parallel with the first worker node rebuild. If it is unsafe to rebuild in parallel because the system could get into a bad state, then it waits to rebuild the second node until it is safe. The `-cmrp` parameter selects the percentage of worker nodes that the worker node rebuild should rebuild at one time. For example, if there are 15 worker nodes and `-cmrp 33` is specified, then 5 worker nodes will be rebuilt at once and with as much parallelization as possible given the state of the system.
```